android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -274,6 +325,31 @@ public void onError(String info) {
         }
     }
 
+    /**
+     * Starts streaming a remote display to the module if there is a connected session. This method of streaming requires the device to be on API level 19 or higher
+     *
+     * @param context            a context that can be used to create the remote display
+     * @param remoteDisplayClass class object of the remote display. This class will be used to create an instance of the remote display and will be projected to the module
+     * @param parameters         streaming parameters to be used when streaming. If null is sent in, the default/optimized options will be used.
+     *                           If you are unsure about what parameters to be used it is best to just send null and let the system determine what
+     *                           works best for the currently connected module.
+     * @param encrypted a flag of if the stream should be encrypted. Only set if you have a supplied encryption library that the module can understand.
+     * @param landscapeRange constraints for vehicle display : aspect ratio, min/max resolutions, max diagonal size.
+     * @param portraitRange constraints for vehicle display : aspect ratio, min/max resolutions, max diagonal size.
+     */
+    public void startRemoteDisplayStream(Context context, Class<? extends SdlRemoteDisplay> remoteDisplayClass, VideoStreamingParameters parameters, final boolean encrypted, VideoStreamingRange landscapeRange, VideoStreamingRange portraitRange) {
```

Comment:
@kostyaBoss Per the proposal, `startRemoteDisplayStream` should be defined as `public void startRemoteDisplayStream(Context context, Class<? extends SdlRemoteDisplay> remoteDisplayClass,
    VideoStreamingParameters parameters, final boolean encrypted, VideoStreamingRange streamingRange)`.

Your definition is different because you take in two `VideoStreamingRanges` instead of just one.  Do you need to change yours to match up with the proposal?

Reply:
@santanamk in the proposal it was just an example provided. To keep libraries mirroring each other we've agreed to make `startRemoteDisplayStream` with two arguments